.github/ISSUE_TEMPLATE/discussion.md

```diff
@@ -0,0 +1,8 @@
+---
+name: Feature Request
+about: Start a discussion for depstat
+title: ''
+labels: kind/discussion
```

Comment:
Please don't use issues for discussions. Discussions should always go to slack, mailing list or discuss.k8s.io